go.mod

```diff
@@ -142,7 +142,7 @@ require (
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
 	github.com/AzureAD/microsoft-authentication-library-for-go v1.1.1 // indirect
-	github.com/BurntSushi/toml v1.2.1 // indirect
+	github.com/BurntSushi/toml v1.3.2 // indirect
```

Comment:
Where this bump is coming from? It's not showing up when locally running a  `go mod tidy && go mod vendor` after having manually updated  `github.com/openshift/assisted-image-service` to `v0.0.0-20240408153851-f822399afa8a`

Reply:
That's how I generated this patch so :man_shrugging: